space/spacecore/storage/sqlitestorage/sql.go

```diff
@@ -0,0 +1,119 @@
+package sqlitestorage
+
+const sqlCreateTables = `
+CREATE TABLE IF NOT EXISTS spaces  (
+  id text not null primary key,
+  header text not null,
+  settingsId text not null,
+  aclId text not null,
+  hash text not null default '',
+  oldHash text not null default '',
+  isCreated boolean not null default false,
+  isDeleted boolean not null default false                        
+);
+
+CREATE TABLE IF NOT EXISTS binds (
+    objectId text not null primary key,
+    spaceId text not null
+);
+
+CREATE TABLE IF NOT EXISTS trees (
+    id text not null primary key,
+    spaceId text not null,
+    type int not null default 0,
```

Comment:
Maybe it should be some kind of enum or constraints?

Reply:
Sqlite doesn't have enums. Constraint I didn't add because we don't have some migration mechanism and will not be easy to change this constraint in the future.